### DISCUSSION TITLE
ORFS verification fails : ERROR: Can't load module `/usr/local/bin/../share/yosys/plugins/eqy_combine.so'

### DISCUSSION THREAD

**Discussion by sandeep-amd** [The-OpenROAD-Project/OpenROAD-flow-scripts#1552] [General]
Steps followed:
Clone:

git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
cd OpenROAD-flow-scripts
sudo ./setup.sh

Build:

./build_openroad.sh --local

Verify:

source ./env.sh
yosys -help
openroad -help
cd flow
make

But here make fails with following errors:
[INFO RSZ-0049] Cloned 2 instances.
[WARNING RSZ-0062] Unable to repair all setup violations.
[INFO RSZ-0033] No hold violations found.
Error: cts.tcl, 110 ERROR: Failed to combine designs. For details see './logs/nangate45/gcd/base/4_eqy_output/combine.log'.
Command exited with non-zero status 1
Elapsed time: 0:03.86[h:]min:sec. CPU time: user 4.00 sys 0.08 (105%). Peak memory: 111068KB.
make[1]: *** [Makefile:683: do-4_1_cts] Error 1
make: *** [Makefile:683: results/nangate45/gcd/base/4_1_cts.odb] Error 2

Log file contents as below,
'./logs/nangate45/gcd/base/4_eqy_output/combine.log':
 Yosys 0.13+15 (git sha1 bc027b2ca, gcc 11.4.0-1ubuntu1~22.04 -fPIC -Os)

-- Executing script file `./logs/nangate45/gcd/base/4_eqy_output/combine.ys' --
ERROR: Can't load module `/usr/local/bin/../share/yosys/plugins/eqy_combine.so': /usr/local/bin/../share/yosys/plugins/eqy_combine.so: undefined symbol: _ZN5Yosys5RTLIL8IdString17destruct_guard_okE

Could you please help with this ?

**Comment by gudeh** [comment]
This was already discussed in other issues/discussions. There is a workaround to manually install binaries from EQY.

**Reply by gudeh** [reply]
#1513

**Comment by vvbandeira** [comment]
@sandeep-amd 
Could you test with the latest code?
